libauk/src/main/java/com/bitmark/libauk/LibAuk.kt

```diff
@@ -31,4 +34,13 @@ class LibAuk {
         val storage = SecureFileStorageImpl(context, uuid)
         return WalletStorageImpl(storage)
     }
+
+    fun calculateFirstEthAddress(words: String, passphrase: String): String {
+        return try {
+            val credential = Bip44WalletUtils.loadBip44Credentials(passphrase ?: "", words);
+            credential.address
+        } catch (e: Exception) {
+            throw e
+        }
+    }
```

Comment:
why not using directly, but using try catch and throw when exception?

btw, passphrase seems to be non optional, so don't need to use `?: ""`